src/App.js

```diff
@@ -100,6 +100,15 @@ const App = (props) => {
     return mailListsData
   }
 
+  const loadInvoiceErrorsData = () => {
+    const invoiceErrorsData =
+      typeof props.errors === 'string' && props.errors !== ''
+        ? JSON.parse(props.errors)
+        : {}
+    console.log(invoiceErrorsData, props)
```

Comment:
Can remove this console log?

Reply:
Is this from a previous commit? :thinking: 
I can't find this console log on that file...